odl/deform/linearized.py

```diff
@@ -85,7 +85,7 @@ def _linear_deform(template, displacement, out=None):
     """
     image_pts = template.space.points()
     for i, vi in enumerate(displacement):
-        image_pts[:, i] += vi.ntuple.asarray()
+        image_pts[:, i] += vi.asarray().ravel()
```

Comment:
And it's wrong but didn't fail because all tests were skipped for some reason (fixture wrong and `available_dtypes` wrong). Fixing.